routes/tasks.js

```diff
@@ -0,0 +1,90 @@
+const express = require('express')
+const router = express.Router()
+const tasksController = require('../controllers/tasksController')
+const { createTask, updateTask } = require('../middlewares/validators/tasks')
+
+/**
+ * @swagger
+ * /tasks:
+ *  get:
+ *   summary: Used to get all the tasks
+ *   responses:
+ *     200:
+ *       description: returns tasks
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/tasks'
+ *     404:
+ *       description : no tasks found
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/errors/notFound'
```

Comment:
I think we don't need it here. We are not returning `404` error response for this request. Can you please remove it?

Thank you! 😄